src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData.ts

```diff
@@ -50,15 +50,10 @@ export type ProteinStats = {
   annotationsCount: number;
 };
 
-export type ProteinSummary = {
-  proteinStats: ProteinStats;
-  pdbeId: string;
-};
-
-export const fetchProteinSummary = async (
+export const fetchXrefId = async (
   proteinId: string,
   signal?: AbortSignal
-): Promise<ProteinSummary | null> => {
+): Promise<Xref | null> => {
```

Comment:
1) Confused about the type signature. With `xrefsData[0]`, isn't it going to be `undefined` rather than `null`?
2) The abort signal doesn't get used at all? Then why does the function want it as a parameter?